draft-ietf-intarea-proxy-config.md

```diff
@@ -357,12 +357,12 @@ This is done to prevent commonly needing to include both "\*.example.com" and "e
 in the `domains` array of a `proxy-match` rule.
 
 The `subnets` array includes IPv4 and IPv6 address literals, as well as IPv4 and IPv6 address subnets
-written using CIDR notation. Subnet-based destination information is only meant be used when applications
-are communicating with destinations identified by only an IP address, and not a hostname.
-
-Destination rules SHOULD NOT contain both the `domains` key and the `subnets` key. Many
-clients will not resolve a domain being accessed through the proxy to an IP address before using
-a proxy, so the subnet information may not be available.
+written using CIDR notation. Subnet-based destination information can apply to cases where
+applications are communicating directly with an IP address (without having resolved a DNS name)
+as well as cases where an application resolved a DNS name to a set of IP addresses. Note that
```

Comment:
Do we care if the all IP from the resolved set are in the subnets or just a subset?